src/azure-cli-core/azure/cli/core/extensions/transform.py

```diff
@@ -5,9 +5,12 @@
 
 import re
 
+from azure.cli.core._util import b64_to_hex
+
 
 def register(application):
     application.register(application.TRANSFORM_RESULT, _resource_group_transform)
+    application.register(application.TRANSFORM_RESULT, _x509_from_base64_to_hex_transform)
```

Comment:
I'm not sure it belongs in core. As far as I can tell, it will only be used in Key Vault.